relay-event-schema/src/protocol/span.rs

```diff
@@ -159,6 +159,8 @@ impl Getter for Span {
                     val.into()
                 } else if let Some(key) = path.strip_prefix("sentry_tags.") {
                     self.sentry_tags.value()?.get(key)?.as_str()?.into()
+                } else if let Some(key) = path.strip_prefix("measurements.") {
+                    self.measurements.value()?.get_value(key)?.into()
```

Comment:
Realized after yesterday that we should insert an extra level here, i.e. `measurements.foo.value` and `measurements.foo.unit`. Just in case we ever want to access the unit in a condition.